packages/core-api/package.json

```diff
@@ -31,6 +31,7 @@
   "dependencies": {
     "@backstage/config": "^0.1.1-alpha.25",
     "@backstage/theme": "^0.1.1-alpha.25",
+    "@backstage/core": "^0.1.1-alpha.25",
```

Comment:
Can't have this here 😅, @backstage/core depends on @backstage/core-api

Reply:
duh 🤦